From 311b503ce2c525080032dd1891248d0850c05060 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:52:59 -0400
Subject: [tools] perf/evsel: Remove duplicate member zeroing after free

Message-id: <1408953210-26343-292-git-send-email-jolsa@redhat.com>
Patchwork-id: 88985
O-Subject: [PATCH RHEL7.1 BZ1133083 291/322] perf evsel: Remove duplicate member zeroing after free
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit de256a4e6b9096070a5305950c7d693395150680
Author: Stephane Eranian <eranian@google.com>
Date: Mon Jan 20 16:15:13 2014 -0300

description
===========
No need to set evsel->fd to NULL after calling perf_evsel__free_fd(), as
this method already does that.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/util/evsel.c b/tools/perf/util/evsel.c
index 22e18a2..55407c5 100644
--- a/tools/perf/util/evsel.c
+++ b/tools/perf/util/evsel.c
@@ -1081,7 +1081,6 @@ void perf_evsel__close(struct perf_evsel *evsel, int ncpus, int nthreads)
 
  perf_evsel__close_fd(evsel, ncpus, nthreads);
  perf_evsel__free_fd(evsel);
- evsel->fd = NULL;
 }
 
 static struct {
-- 
1.7.1